### ISSUE TITLE
It does not render when using latest react-redux version

### ISSUE THREAD

**Issue by leosbrf** [eclipsesource/jsonforms#1189]
**Describe the bug**
It not renders and throws this error:
```
Could not find "store" in either the context or props of "Connect(JsonFormsDispatchRenderer)". Either wrap the root component in a <Provider>, or explicitly pass "store" as a prop to "Connect(JsonFormsDispatchRenderer)"
```

**To Reproduce**
1. Follow the tutorial on [JsonForms Website](https://jsonforms.io/docs/tutorial)
2. Start the application and everything works fine
3. Now add the last react-redux version (`yarn add react-redux`)
4. Start the application and it throws the error above

**Expected behavior**
It should render.

**Screenshots**
![image](https://user-images.githubusercontent.com/9287342/49703365-3e8b4100-fbeb-11e8-87d8-999dd11d5288.png)

**Browser (please complete the following information):**
 - Chrome
 - Version 70.0.3538.110

**Used Setup (please complete the following information):**
 - Framework: react 16.6.3
 - redux: 4.0.1
 - react-redux: 6.0.0
 - jsonforms/core: 2.0.12
 - jsonforms/material-renderers: 2.0.12
 - RendererSet: material

**Additional context**
N/A

**Comment by edgarmueller** [comment]
Thanks for the report, we'll look into it.

**Comment by zummed** [comment]
Seems to be caused by a clash between a projects react-redux dependency, and jsonforms nested older version of react-redux.  

A temporary work-around if using "yarn" is to override all versions of react-redux by putting the following in your package.json:

```
...
"resolutions": {
    "react-redux": "^6.0.0"
}
...
```

**Comment by edgarmueller** [comment]
We updated the dependency with #1221, which should hopefully fix the issue. It'll be part of the 2.2.0 release.

**Comment by edgarmueller** [comment]
On a different note: we should probably make `react-redux` a peer dependency, which is part of issue #1230

**Comment by clayroach** [comment]
@edgarmueller - this one cropped back up for us after the upgrade to `2.1.1-alpha.3`.  

Attempted workarounds:
Removing package-lock.json & changing `react-redux: "^6.0.0"` in our package.json didn't resolve.

**Comment by edgarmueller** [comment]
@clayroach Sorry to head that. Did you also try removing the node_modules folder? I guess we need to address #1230 first in order to get rid of these versioning problems. We'll delay the 2.2.x release until these issues are fixed.

**Comment by clayroach** [comment]
Yep - tried to remove node folders, lock file, run clean builds, etc, etc.  I think #1230 is in order to resolve this one.

**Comment by edgarmueller** [comment]
We've looked into the issue and configured our peer dependencies accordingly today (#1245). A new testing release is available as 2.1.1-alpha.4. We didn't really have much time to test it thoroughly though, so it would be really great if you could give this a try and report any issues. We've tested it by using create-react-app 2.1.1 in order to re-create the JSONForms tutorial app, which did work.

On a different note: Please note that we are stuck with redux 3 currently because of the Angular-based renderer sets (see #1243 & #1244). We'll will tackle updating to redux 4 after 2.2.0 has been released.

**Comment by clayroach** [comment]
@edgarmueller Looks like 2.1.1-alpha.4 resolves this issue.
